src/plugins/messenger/MessengerPreview/components/MessengerGenericTemplate/MessengerGenericTemplate.tsx

```diff
@@ -113,9 +116,13 @@ export const getMessengerGenericTemplate = ({
                             onClick={e => default_action && onAction(e, default_action)}
                             className="webchat-carousel-template-content"
                             style={default_action ? { cursor: "pointer" }:{}}
+                            role={default_action ? "link" : undefined}
```

Comment:
the "default action" wasn't keyboard accessible for me in MS Edge (>80), couldn't focus it without a mouse

Reply:
Are you sure you weren't able to focus it using keyboard? This should not happen in any browser, as the item has a tabindex 0. If you mean that you could not open the link on 'Enter' keypress, then I have fixed it by adding onKeyDown handler.

Reply:
For me is also working on MS Edge (>80)

Reply:
works for me, will resolve this comment